content/applications/finance/payment_providers.rst

```diff
@@ -169,246 +162,267 @@ Bank payments
+:guilabel:`Configuration` tab.
 
-.. tip::
-   As there might be errors, updates, and unknowns in the lists of available currencies, it is
-   possible to add or remove currencies.
+.. _payment_providers/currencies_countries:
 
-If the list of supported currencies is empty, it either means the list is too long to be displayed,
-or Odoo does not have information on that payment provider. The payment provider remains available,
-even though it is possible the payment will be refused at a later stage should the currency not be
-supported. This field is required if the user needs to select a currency when setting up their
-payment provider account.
+Currencies and countries
+------------------------
 
-.. _payment_providers/countries:
+All payment providers have a different list of available currencies and countries. They serve as a
+first filter during payment operations, i.e., the payment methods linked to the payment provider are
+not available for selection if the customer's currency or country is not in the supported list. As
+there might be errors, updates, and unknowns in the lists of available currencies and countries, it
+is possible to add or remove a payment provider's supported currencies or countries.
 
-Countries
-*********
+.. note::
 
-Restrict the use of the payment provider to a selection of countries. Leave this field blank to make
-the payment provider available in all countries.
+   - :ref:`Payment methods <payment_providers/payment_methods>` also have their own list of
+     available currencies and countries that serves as a filter during payment operations.
+   - If the list of supported currencies or countries is empty, it either means the list is too long
+     to be displayed, or Odoo does not have information on that payment provider. The payment
+     provider remains available, even though it is possible the payment will be refused at a later
+     stage should the currency not be supported.
 
-Maximum Amount
-**************
+Maximum amount
+--------------
 
-Restrict the maximum amount that can be paid with the selected provider. Leave this field to `0.00`
-to make the payment provider available regardless of the payment amount.
+You can restrict the :guilabel:`Maximum Amount` that can be paid with the selected provider. Leave
+the field to `0.00` to make the payment provider available regardless of the payment amount.
 
 .. important::
-   This feature is not intended to work on pages which allow the customer to update the payment
-   amount. For example, the **Donation** snippet of the Website app, and the **Checkout** page of
-   the **eCommerce** app when paid delivery methods are enabled.
+   This feature is not intended to work on pages that allow the customer to update the payment
+   amount, e.g., the **Donation** snippet and the **Checkout** page when paid :doc:`shipping methods
+   <../websites/ecommerce/checkout_payment_shipping/shipping>` are enabled.
 
 .. _payment_providers/journal:
 
 Payment journal
```

Comment:
Do what you want with this information: payment journals must be configured only if Invoicing/Accounting is installed, since Odoo 17.